data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1850 @@
+				decline
+			label metchiree
+			`More blue-uniformed workers are waiting when you land, though these uniforms consist of incredibly thick, bulky coats. They take the crates out of your ships while the "doctors" are directed to a hospital building, and <payment> are transferred to you when the workers are done. They thank you for your help, and come in close as they're about to leave, quietly asking that you look for more members on worlds on northern Kimek space.`
+
+mission "Resistance: Smuggling: Charity 3"
+	name "Charity Delivery"
+	description "Bring <cargo> to <destination>"
+	cargo "charity supplies" 46
+	source
+		near "14 Pole" 3 5
+	destination "Remote Blue"
+	to offer
+		random < 70
+		has "Resistance: Smuggling: Charity 2: done"
+		not "Heliarch Investigation 2: active"
+		not "Heliarch License: done"
+		not "Resistance: Questions: active"
+	on offer
+		conversation
+			branch metchiree
+				has "Resistance: Questions: done"
+			`A group of Kimek approach you in the spaceport. They are all in the blue uniforms of the charity you have been helping by transporting supplies and people. "Greetings, Captain <last>," one of them says. "Helped our charity before, you have. Interested in more work, are you?"`
+			choice
+				`	"What do you need me to do?"`
+				`	"Sorry, I'm not interested in helping anymore."`
+					decline
+			`	"Lack many of their preferred diet options, the Arachi and Saryds on <planet> do," they explain. "Enough to order it from off-world, the poorer of them do not have, so sending them a care package, our group is."`
+			`	They show you to a pile of containers and crates near the hangars, and you have them load them into your ship. "Once on <planet>, you are, pay you <payment>, we will. Thank you for the help, Captain." They wish you a safe trip, and vanish into the spaceport halls.`
+				accept
+			label metchiree
+			`A group of Kimek approach you in the spaceport. They are all in the blue uniforms of Chiree's "charity" group, and one of them signals to your ship once they're close. You get the hint, and bring them aboard. "More work for you, Chiree has. To <planet>." They begin the moment you close the hatch. "Prepared an important job, she has. One only you can do. Paid <payment> to deliver <cargo> to <planet>, you will be, and once there, tell you the details about the main mission, she can."`
+			choice
+				`	"Understood. I'll go there right away."`
+				`	"I'm sorry, but I don't think this line of work is for me."`
+					decline
+			`	They nod, and you all leave your ship. They show you a pile of containers and crates near the hangars, and load them in your cargo hold.`
+				accept
+	on complete
+		payment 261000
+
+mission "Resistance: Smuggling: Grenades"
+	name "Grenade Smuggling"
+	description "Bring 100 fragmentation grenades from human space to <destination>."
```

Comment:
Since these missions function by taking actual outfits upon completion rather than assigning a `cargo` field and providing that to the player, I don't think so.